cmd/experimental/kjobctl/docs/commands/kjobctl_crds/_index.md

```diff
@@ -0,0 +1,231 @@
+<!--
+The file is auto-generated from the Go source code of the component using the
+[generator](https://github.com/kubernetes-sigs/kueue/tree/main/cmd/experimental/kjobctl/hack/tools/kjobctl-docs).
+-->
+
+# kjobctl crds
+
+
+## Synopsis
+
+
+Print the kjobctl CRDs
+
+```
+kjobctl crds [flags]
+```
+
+
+## Examples
+
+```
+  # Install or update the kjobctl CRDs 
+  kjobctl crds | kubectl apply --server-side -f -
```

Comment:
"Crds" without a verb is ambiguous. Can we call it printcrd or alike to indicate that this command just print crds and extra piping is needed?

Reply:
done